### PR TITLE
Bootstrap 3 Migration: Edit Feature Flag Page

### PR DESCRIPTION
**Pull request by pnaw94**
@biyeun 

screenshot01
screenshot02

### CODE REVIEW

File: corehq/apps/toggle_ui/templates/toggle/edit_flag.html [hunk 1]
```diff
@@ -6,10 +6,9 @@
 {% block title %}{% trans "Edit Feature Flag: " %}{{ toggle_meta.label }}{% endblock %}
 
 {% block js %}{{ block.super }}
-    <script src="{% static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
-    <script src="{% static 'hqwebapp/js/main.js' %}"></script>
-    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
-    <script src="{% static 'formdesigner/js/lib/underscore-1.5.2.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/ajax_csrf_setup.js' %}"></script>
+    <script src="{% new_static 'hqwebapp/js/main.js' %}"></script>
+    <script src="{% new_static 'formdesigner/js/lib/underscore-1.5.2.js' %}"></script>
```
Comment (benrudolph) [inline corehq/apps/toggle_ui/templates/toggle/edit_flag.html]: could we use the bower underscore here? `underscore/underscore.js`

Comment (benrudolph) [inline corehq/apps/toggle_ui/templates/toggle/edit_flag.html]: actually i think underscore is always loaded on the page so we could probably get rid of it entirely

File: corehq/apps/toggle_ui/templates/toggle/edit_flag.html [hunk 2]
```diff
@@ -138,21 +136,21 @@
         </div>
         <hr/>
         <!-- ko foreach: items -->
-        <div class="row">
-            <div class="span1">
-                <a href="#" class="btn btn-danger" data-bind="click: $parent.removeItem"><i class="icon-trash"></i></a>
+        <div class="row" style="margin: 5px 0">
```
Comment (biyeun) [inline corehq/apps/toggle_ui/templates/toggle/edit_flag.html]: the delete button and the input should both be inline in the same column. Also, let's stay away from using inline styles

**Comment by biyeun** [comment]
just one comment and that's it

**Comment by pnaw94** [comment]
Updated

**Comment by biyeun** [comment]
fab :+1:

**Pull request merged**